sound/soc/intel/atom/sst/sst_stream.c

```diff
@@ -174,10 +174,11 @@ int sst_send_byte_stream_mrfld(struct intel_sst_drv *sst_drv_ctx,
 	u32 length;
 	int pvt_id, ret = 0;
 	struct sst_block *block = NULL;
+	u8 bytes_block = bytes->block;
```

Comment:
if this is a false positive, can we ignore it?

Reply:
Isn't a local variable clearer and better anyway? Faster and less concurrency.